terraform/chef-server.sh

```diff
@@ -71,7 +51,7 @@ password = 'chefrocks'
   --user="${USER_ID:-42}:${GROUP_ID:-42}" \
   --network=host \
   --detach=true \
-  ${DOCKER_ORIGIN:-chefserverofficial}/postgresql:${VERSION:-latest}
+  ${CHEF_SERVER_DOCKER_ORIGIN:-chefserverofficial}/postgresql:${CHEF_SERVER_VERSION:-stable}
```

Comment:
any problem with us switching this to `$AUTOMATE_DOCKER_ORIGIN` and save us from publishing and independently testing this package?   same for elasticsearch5 below

Reply:
Will do. Just a note: also requires changing to `$AUTOMATE_VERSION`